apps/nextjs-website/src/components/organisms/WebinarQuestionsTemplate/WebinarQuestionsTemplate.tsx

```diff
@@ -69,24 +72,28 @@ const WebinarQuestionsTemplate = ({
                 </TableRow>
               </TableHead>
               <TableBody>
-                {sortedQuestions.map((row) => (
+                {sortedQuestions.map((webinarQuestion) => (
                   <WebinarQuestionRow
-                    key={row.createdAt.toJSON()}
-                    question={row}
+                    key={webinarQuestion.createdAt.toJSON()}
```

Comment:
```suggestion
                    key={webinarQuestion.createdAt.toISOString()}
```
I think that function is clearer than the `toJSON`. And [the output is the same](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toJSON)